[@nativescript/picker] build.all doesn't build @nativescript/picker plugin

After selecting `@nativescript.build-all` all of the plugins are starting to build except `@nativescript/picker`.

Error message after fresh build:
`npm ERR! Could not install from "../../dist/packages/picker" as it does not contain a package.json file.`

--- a/workspace.json
+++ b/workspace.json
@@ -221,7 +221,8 @@
 							"nx run auto-fit-text:build.all",
 							"nx run animated-circle:build.all",
 							"nx run localize:build.all",
-							"nx run detox:build.all"
+							"nx run detox:build.all",
+							"nx run picker:build.all"
 						],
 						"parallel": false
 					}

